test/Confluent.Kafka.UnitTests/Confluent.Kafka.UnitTests.csproj

```diff
@@ -17,4 +17,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Don't think we want this:
http://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for